scripts/autotools/libmongocrypt/Version.m4

```diff
@@ -0,0 +1,6 @@
+if test "$PHP_CLIENT_SIDE_ENCRYPTION" != "no"; then
+  MONGOCRYPT_CURRENT_FILE=[]PHP_EXT_SRCDIR(mongodb)[/src/LIBMONGOCRYPT_VERSION_CURRENT]
+  BUILD_VERSION=$(cat $MONGOCRYPT_CURRENT_FILE)
+
+  AC_SUBST(BUILD_VERSION)
```

Comment:
I think [MONGOCRYPT-220](https://jira.mongodb.org/browse/MONGOCRYPT-220) is merged, so this can be changed. If not, let's add a TODO.